### PR TITLE
Fixed sentences that had awkward translation

### PR DESCRIPTION
**Pull request by bingdai**

### CODE REVIEW

File: docs/docs/01-why-react.zh-CN.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-React 是一个 Facebook 和 Instagram 用来创建用户界面的 JavaScript 库。很多人选择将 React 认为是 **[MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller)** 中的 **V**（视图）。
+React 是一个已被 Facebook 和 Instagram 所采用的、用于创建用户界面的 JavaScript 库。很多人认为 React 是 **[MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller)** 中的 **V**（视图）。
 
-我们创造 React 是为了解决一个问题：**构建随着时间数据不断变化的大规模应用程序**。
+我们创造 React 的目的是解决这样一个问题：**构建随着时间数据不断变化的大规模应用程序**。
 
 ### 简单
 
-仅仅只要表达出你的应用程序在任一个时间点应该呈现的样子，然后当底层的数据变了，React 会自动处理所有用户界面的更新。
+你仅仅只需要表达出你的应用程序在任一个时间点应该呈现的样子，当底层的数据变了时，React 会自动更新所有用户界面。
```
Comment (keyanzhang) [inline docs/docs/01-why-react.zh-CN.md]: How about "仅需要表达出你的应用程序在任何时间应该如何呈现，当底层的数据变化时，React 会自动更新所有用户界面。"?

File: docs/docs/01-why-react.zh-CN.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
 ### 声明式 (Declarative)
 
-数据变化后，React 概念上与点击“刷新”按钮类似，但仅会更新变化的部分。
+数据变化后，React 概念上与点击“刷新”按钮类似，但仅会更新应当变化的部分。
 
 ## 构建可组合的组件
 
-React 都是关于构建可复用的组件。事实上，通过 React 你*唯一*要做的事情就是构建组件。得益于其良好的封装性，组件使代码复用、测试和关注分离（separation of concerns）更加简单。
+用React 所构建的组件都是可复用的。事实上，在采用 React 时你*唯一*要做的事情就是构建组件。得益于其良好的封装性，组件使代码在复用、测试和关注分离（separation of concerns）等方面都更加简单。
```
Comment (keyanzhang) [inline docs/docs/01-why-react.zh-CN.md]: “使用” might be better than “采用”。There is a whitespace missing before the first "React". :)

**Comment by keyanzhang** [comment]
ping.

**Comment by zpao** [comment]
@keyanzhang if this is fine and just needs a bit of followup, can you create a new PR keeping the original commit and your adjustments in a 2nd commit?

**Comment by keyanzhang** [comment]
@zpao Sure.

**Comment by gaearon** [comment]
Thanks for the PR!

We just released [new docs mostly written from scratch](https://facebook.github.io/react/). However this means that all old localized content is gone, as new one needs to be written.

We will track this effort in https://github.com/facebook/react/issues/8063.

Sorry, and thank you for your work!

**Pull request closed without merging**